src/Fixer/Whitespace/StatementIndentationFixer.php

```diff
@@ -198,12 +208,25 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
 
         $previousLineInitialIndent = '';
         $previousLineNewIndent = '';
+        $noBracesBlockStarts = [];
```

Comment:
overall, PER-CS requires ([ref](https://www.php-fig.org/per/coding-style/#5-control-structures)) to have the braces.

I'm actually OK to not have braces-less variant not supported.
before we merge, is there any community that benefits from not having the braces?

Reply:
IMO this should be supported because the intent of the rule is to indent all statements, not only statements in code that follows state of the art code styles. Also the added maintainance cost is not critical as opposed to e.g. support for PHP files with mixed HTML.

However, not having braces is widely considered a bad practice so I don't expect it to be part of any recommendation and I would understand you don't want to merge this.

Reply:
> Also the added maintainance cost is not critical as opposed to e.g. support for PHP files with mixed HTML.

we don't support mixed HTML ;)

-----

> However, not having braces is widely considered a bad practice so

So that's where i wonder. If there is strong community that is _not_ using `{}` for all the cases, it makes sense to add the support.

if there is no strong community of _not_ using `{}`, I do not see huge value to have it supported and rather recommend individuals to start using `{}`, but I would also not block this PR to get merged. Your decision here!

Reply:
Alright then I propose to merge this when ready because
> […] the intent of the rule is to indent all statements

but only if fixing [this case](7663#discussion_r1439769172) does not require too much complicated changes.

Reply:
Changes size increased but remains acceptable to me. What do you think?